db/migrate/20211208095206_add_multiple_index_for_answers.rb

```diff
@@ -0,0 +1,8 @@
+class AddMultipleIndexForAnswers < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+  
+  def change
+    add_index :answers, [:question_id, :created_at], algorithm: :concurrently
+    safety_assured { remove_column :results, :time_duration, :integer }
```

Comment:
I can't understand why when i run migration, created time_duration column, so i removed this column